src/schema/updates/manifest.json

```diff
@@ -82,6 +82,9 @@
           "version": {
             "description": "Version string must be a string comprising one to four dot-separated integers (0-65535). E.g: 1.2.3.",
             "format": "versionString"
+          },
+          "manifest_version": {
+            "maximum": 2
```

Comment:
Do we have to remember adding this manually every time we import the schema? If so, that seems risky.

Reply:
This looks like a manual override (opposed to the files in `imported`), and there's a unit test that would fail if this constraint were to be dropped. Therefore I have confidence in the direction of this patch.

Luca can confirm if needed.

Reply:
@wagnerand no, we don't need to remind overselves to reapply the override, that is what the change in src/schema/updates/manifest.json does (it gets propagate on every schema import).

I did put an inline comment nearby the test case to remind ourselves that we will have to remove that override when we are ready to accept manifest_version 3 extensions on AMO (we may also file an issue to keep open and remind that as well).

Reply:
Thank you both for clarifying!